back/KinoFund/Core/Dtos/Comments/CreateCommentDTO.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Core.Dtos.Comments
+{
+    public class CreateCommentDTO
+    {
+        public long UserId { get; set; }
+        public long MovieId { get; set; }
+        public long? RefersTo { get; set; }
+        public DateTime Date { get; set; }
```

Comment:
you don't need Date field  - you send this field on backend side (DateTime.UtcNow)